### DISCUSSION TITLE
Projects (beta): autofilled fields from a linked PR

### DISCUSSION THREAD

**Discussion by Totktonada** [community/community#5539] [Projects and Issues]
I would like to create a 'Needs review from <...>' view with grouping by a reviewer. Since we anyway set reviewers in PRs and link PRs to issues, it would be nice to autofill a field with a reviewers in a project table.

I would highlight that we often has two reviewers and the grouping should include such issue in all groups (as for the first reviewer as well as for the second).

**Comment by whoaitsaimz** [comment]
Would love to see Issue linked to PR in Projects beta automatically incl. review status.
